lib/shopify_api/availability/rest.ex

```diff
@@ -0,0 +1,16 @@
+defmodule ShopifyAPI.Availability.REST do
```

Comment:
Is `Availability` a better namespacing choice than `RateLimiting`?

Reply:
We took the naming cue from the existing `AvailabilityTracker`. I have no strong feelings one way or the other, @hez?

Reply:
It was just a cue from the existing code, the age old naming things problem....

- `Availability`
- `RateLimiting`
- `Throttling`

Any other contestants?

Reply:
`RateLimiting` it is!